### PR TITLE
[TW-402] Reorganize audit logs

### PR DESCRIPTION
**Pull request by claireriley**

### CODE REVIEW

File: src/pages/docs/administration/audit-logs.md [hunk 1]
```diff
@@ -80,6 +80,22 @@ Postman logs events related to [team management](#team-management), [user manage
...
 | Deleted Custom Domain  | A custom domain was deleted from your team. |
+| Custom Auth Scheme Enabled | An SSO scheme was enabled.  |
+| Custom Auth Scheme Updated  | An SSO scheme was updated. |
+| Custom Auth Scheme Disabled  | An SSO scheme was disabled. |
+| Custom Auth Scheme Removed  | An SSO scheme was removed.  |
+| Created SCIM Key | A user created a new SCIM key. |
+| Activated SCIM Key | A user activated a SCIM key. |
+| Regenerated SCIM Key | A user regenerated an SCIM key. |
+| Revoked SCIM Key | A user revoked an SCIM key. |
+| Deleted SCIM Key | A user deleted an SCIM key. |
```
Comment (HeatherStenson) [inline src/pages/docs/administration/audit-logs.md:91]:
```suggestion
| Regenerated SCIM Key | A user regenerated a SCIM key. |
| Revoked SCIM Key | A user revoked a SCIM key. |
| Deleted SCIM Key | A user deleted a SCIM key. |
```

**Review by HeatherStenson — APPROVED** [review]
Looks good to me! Just one suggestion (`an` --> `a`) 🍊

**Review by mark-anthony-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/administration/audit-logs.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ contextual_links:
 
 > __[Audit logs are available on Postman Professional and Enterprise plans.](https://www.postman.com/pricing)__
 
-Audit logs display events related to team management, billing, and security. Team admins can review audit logs to determine:
+Audit logs display events related to team management, user management, and billing. Team admins can review audit logs to determine:
```
Comment (ronakodhaviya) [inline src/pages/docs/administration/audit-logs.md:17]: @claireriley could we remove the 'management' word and simply say

> Audit logs display events related to the team, users, and billing.

I found this to be a general practice among SaaS tools with audit log support, for example, Slack uses the same structure. Ref: https://api.slack.com/admins/audit-logs#audit_logs_actions

Let me know what you think :)

**Pull request merged**